apps/api/src/middleware/authenticateToken.ts

```
@@ -0,0 +1,27 @@
+import jwt, { JsonWebTokenError, TokenExpiredError } from "jsonwebtoken";
+import { TUser } from "../models/User";
+
+interface Request extends ExpressRequest {
+  user?: TUser;
+}
+
+const JWT_SECRET = process.env.JWT_SECRET as string;
+
+export const authenticateToken = (req: Request, res: Response, next: NextFunction) => {
+  const authHeader = req.headers.authorization;
+  const token = authHeader && authHeader.split(" ")[1];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)